Apps/Common/SIMSolverAdap.h

```diff
@@ -68,7 +100,10 @@ template<class T1> class SIMSolverAdap : public SIMSolverStat<T1>
     return aSim.parse(elem);
   }
 
-  AdaptiveSIM aSim; //!< Adaptive simulation driver
+  AdapSim aSim; //!< Adaptive simulation driver
 };
 
+template<class T1>
+using SIMSolverAdap = SIMSolverAdapImpl<T1, AdaptiveSIM>; //!< Convenience typedef
```

Comment:
The doxy is inaccurate, this is not a typedef, but... something else I haven't seen. Is it equivalent to
`template<class T> class SIMSolverAdap : public SIMSolverAdapImps<T,AdaptiveSIM> {};`

Reply:
it's a c++11 style typedef, if you want to be nitpicky the exact name is an alias template http://en.cppreference.com/w/cpp/language/type_alias

it was added to the language precisely to avoid the need for such wrapper templates so yes.

Reply:
right, perfect. I now see similar constructs in AdvectionDiffusion.h, for instance. There it is documented as "Convenience rename" so only a bit in-consequent.